search_gov_crawler/requirements.txt

```diff
@@ -1,7 +1,9 @@
+logparser==0.8.2
```

Comment:
why did you include this here? isn't it bundled with scrapydweb?

Reply:
You are right, it's not needed